Havit.Blazor.Components.Web.Bootstrap/Forms/InputSettings.cs

```diff
@@ -9,4 +9,8 @@ public record InputSettings
 	/// Specifies how the validation message should be displayed.
 	/// </summary>
 	public ValidationMessageMode? ValidationMessageMode { get; set; }
+	/// <summary>
+	/// Specifies how the label should be displayed.
+	/// </summary>
+	public LabelType? LabelType { get; set; }
```

Comment:
The current issue is that not all input components have `LabelType` (implement `IInputWithLabelType`), which is why we left this unimplemented and didn't allow setting `LabelType` from a single place.

However, I understand there's a valid need to have a central point to set `LabelType`, even if some input components don't support it. I'll do some rework here...